WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/OrderCustomerSummary.swift

```diff
@@ -0,0 +1,108 @@
+import SwiftUI
+
+struct OrderCustomerSummary: View {
+
+    let viewModel: NewOrderViewModel.CustomerDataViewModel
+
+    @Environment(\.presentationMode) var presentation
+    @Environment(\.safeAreaInsets) var safeAreaInsets: EdgeInsets
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+                Spacer(minLength: Layout.sectionSpacing)
+                contactsSection(title: viewModel.fullName, subtitle: viewModel.email)
+                Spacer(minLength: Layout.sectionSpacing)
+                addressSection(title: Localization.shippingAddressSection, address: viewModel.shippingAddressFormatted)
+                Spacer(minLength: Layout.sectionSpacing)
+                addressSection(title: Localization.billingAddressSection, address: viewModel.billingAddressFormatted)
+            }
+        }
+        .background(Color(.listBackground).ignoresSafeArea())
+        .ignoresSafeArea(.container, edges: [.horizontal])
+        .navigationTitle(Localization.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .cancellationAction) {
+                Button(Localization.closeButton) {
+                    presentation.wrappedValue.dismiss()
+                }
+            }
+        }
+        .wooNavigationBarStyle()
+    }
+}
+
+private extension OrderCustomerSummary {
+    @ViewBuilder func contactsSection(title: String?, subtitle: String?) -> some View {
+        ListHeaderView(text: Localization.contactSection, alignment: .left)
+            .padding(.horizontal, insets: safeAreaInsets)
+        Group {
+            Divider()
+            NavigationRow(content: {
+                VStack(alignment: .leading, spacing: Layout.verticalEmailSpacing) {
+                    if let title = title {
+                        Text(title)
+                            .bodyStyle()
+                    }
+                    if let subtitle = subtitle {
+                        Text(subtitle)
+                            .footnoteStyle()
+                    }
+                }
+            }, action: {})
+            Divider()
+        }
+        .background(Color(.listForeground))
+    }
+
+    @ViewBuilder func addressSection(title: String, address: String?) -> some View {
+        ListHeaderView(text: title, alignment: .left)
+            .padding(.horizontal, insets: safeAreaInsets)
+        Group {
+            Divider()
+            NavigationRow(content: {
+                Text(address ?? Localization.emptyAddressPlaceholder)
```

Comment:
I'm never seeing this placeholder, even if there isn't any address information to display:

<img src="https://user-images.githubusercontent.com/8658164/148414858-d4ec3f5d-d4df-4e7f-ae1a-c0778b20d68d.png" width="300px">

It looks like `address` is an empty string (instead of `nil`) so we need to check for that case to show the placeholder.